packages/angular/ssr/tokens/BUILD.bazel

```diff
@@ -0,0 +1,19 @@
+load("//tools:defaults.bzl", "ts_library")
+
+package(default_visibility = ["//visibility:public"])
+
+ts_library(
+    name = "tokens",
+    srcs = glob(
+        [
+            "*.ts",
+            "src/**/*.ts",
+        ],
+    ),
+    module_name = "@angular/ssr/tokens",
```

Comment:
Any other ideas for as an entry-point name?

Reply:
`@angular/ssr/tokens` seems reasonable to me.